package com.knittingstories.user.dto;

public record AuthResponse(
        String token,
        UserResponse user
) {
}
